google-cloud-firestore/src/test/java/com/google/cloud/firestore/TransactionTest.java

```diff
@@ -428,6 +575,49 @@ public void getMultipleDocumentsWithFieldMask() throws Exception {
     assertEquals(commit(TRANSACTION_ID), requests.get(2));
   }
 
+  @Test
+  public void getMultipleDocumentsWithFieldMaskAsync() throws Exception {
```

Comment:
Your PRs changes are already tested by `getDocumentAsync()`. Since there is  no specific handling of GetAll, I would suggest to remove this test.